service-api/module/Application/test/ApplicationTest/Services/SessionStatusServiceTest.php

```diff
@@ -288,7 +288,11 @@ public function testPassportValidityCheckIfUKPassportUsed(): void
     {
         $caseData = CaseData::fromArray([
             'id' => '2b45a8c1-dd35-47ef-a00e-c7b6264bf1cc',
-            'idMethod' => 'po_ukp',
+            "idMethodIncludingNation" => [
+                'id_method' => "PASSPORT",
+                'id_country' => "GBR",
+                'id_route' => "TELEPHONE",
```

Comment:
It doesn't particularly matter but for the sake of avoiding confusion all the examples in this file should have POST_OFFICE as their route.